oak_containers/proto/interfaces.proto

```diff
@@ -41,9 +42,15 @@ service Launcher {
   // Provides orchestrator with the trusted container image.
   rpc GetContainerBundle(google.protobuf.Empty) returns (stream GetImageResponse) {}
 
-  // This could be anything that the application wants to attest to.
-  // e.g. whether it's in debug mode, or parameters for aggregation / DP,
-  // or digest of something else that the application will load later (e.g. Wasm module).
-  // It may be empty if the application does not require one
+  // This method is used by the orchestartor to load and measure the
+  // application config. The orchestrator will later, separately expose this
```

Comment:
Would be nice to specifiy "Trusted Application config" in the comment---maybe even in the method name?

Reply:
Good call, adding it in the comments for now. We can update the method name in another PR